packages/create-indiekit/files/template.vscode_launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Indiekit",
+      "skipFiles": ["<node_internals>/**"],
+      "program": "${workspaceFolder}/node_modules/@indiekit/indiekit/bin/cli.js",
+      // "args": ["--config", "path/to/your/indiekit.config.js"]
```

Comment:
Is this commented out code needed?

Reply:
I wasn't sure which arguments were accepted by `cli.js`. Later I ended up using this:

```json
"args": ["--config", "indiekit.config.js", "serve"]
```